js/foundation.dropdown.js

```diff
@@ -150,14 +150,14 @@ class Dropdown extends Positionable {
         if (_this.options.forceFollow === false) {
           // if forceFollow false, always prevent default action
           e.preventDefault();
-        } else if (hasTouch && _this.options.hover && _this.$element.hasClass('is-open') === false) {
+        } else if (hasTouch && _this.options.hover !== false && _this.$element.hasClass('is-open') === false) {
           // if forceFollow true and hover option true, only prevent default action on 1st click
           // on 2nd click (dropown opened) the default action (e.g. follow a href) gets executed
           e.preventDefault();
         }
     });
 
-    if(this.options.hover){
+    if(this.options.hover !== false){
```

Comment:
With `<div data-dropdown data-hover></div>`, `this.options.hover` is equal to an empty string here. So I wonder, do we need to:
- consider all non-false values as positives, which is a classic HTML approach
- ignore all invalid properties (expect a boolean but got a string), which is more sustainable approach.

The problem here is that we work with both HTML and Js values, and that in HTML `attribute` equals to `attribute=""`. Also, HTML boolean attibutes as considered as `true` as soon as they are set, even with a `"false"` as value...

With the current PR, `Foundation.Dropdown.defaults.hover = null` will make it considered as `true` even if its default is `false`.

Reply:
So check better for strict true or any type of string?